GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -86,5 +86,26 @@ private void DirectoryKeyPress(object sender, KeyPressEventArgs e)
                 LoadClick(null, null);
             }
         }
+
+        private void folderGoUpbutton_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                var currentFolder = new DirectoryInfo(_NO_TRANSLATE_Directory.Text);
+                if (currentFolder.Exists)
```

Comment:
perhaps it worth indicating it to the user that folder doesn't exist. 
for example, by disabling controls or by showing a message